mods/sethome/init.lua

```diff
@@ -54,11 +54,13 @@ minetest.register_chatcommand("sethome", {
         minetest.chat_send_player(name, "Home set!")
         changed = true
         if changed then
-        	local output = io.open(homes_file, "w")
-            for i, v in pairs(homepos) do
-                output:write(v.x.." "..v.y.." "..v.z.." "..i.."\n")
-            end
-            io.close(output)
+        	local output = io.open(home_file, "w")
+			local data = {}
```

Comment:
@pinkysnowman imo, it would be better to leave a reply for the pull request on the pull request rather than in the note